druid/examples/input_region.rs

```diff
@@ -0,0 +1,194 @@
+use druid::widget::prelude::*;
```

Comment:
This file needs header comments (copyright/description), take a look at how other examples do it.

Reply:
Added.